php-frontend/src/main/java/org/sonar/php/tree/symbols/QualifiedNames.java

```diff
@@ -20,14 +20,23 @@
 package org.sonar.php.tree.symbols;
 
 import org.sonar.plugins.php.api.symbols.QualifiedName;
+import org.sonar.plugins.php.api.symbols.Symbol;
 
 public class QualifiedNames {
 
   private QualifiedNames() {
   }
 
+  /**
+   * @deprecated since 3.14 - Use {@link #memberName(QualifiedName, String, Symbol.Kind)} instead
+   */
+  @Deprecated
```

Comment:
`QualifiedNames` is not part of the public API because it's not in a subpackage of `org.sonar.plugins.php.api`. I think we can drop this method.